docs/manuals/core/concepts/pipeline.md

```diff
@@ -17,7 +17,7 @@ preprocessing and preparing data, a _pipeline_ for computing a training model, a
 
     This problem has been modeled in two pipelines - one pipeline for the forecasting algorithm and one for the
     production planning algorithm. As a consequence, the two algorithms can have two different workflows. They can run
-    independently, under different schedules. For example, one on a fixed schedule (e.g. every week) and one on demand,
+    independently, under different orchestrations. For example, one on a fixed orchestration (e.g. every week) and one on demand,
```

Comment:
In this specific case, we are not talking about orchestration but schedule. We are comparing two workflow schedules...

```suggestion
    independently, under different schedules. For example, one on a fixed schedule (e.g. every week) and one on demand,
```